tests/neptune/new/attributes/test_attribute_base.py

```diff
@@ -43,7 +45,7 @@ def _create_run(processor: Optional[OperationProcessor] = None):
         backend = NeptuneBackendMock()
```

Comment:
Do we have a task for testing this with `ContainerType.PROJECT`?

Reply:
No